frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx

```diff
@@ -294,12 +331,17 @@ export const ClusterServiceVersionsDetailsPage: React.StatelessComponent<Cluster
     namespace={props.match.params.ns}
     kind={referenceForModel(ClusterServiceVersionModel)}
     name={props.match.params.name}
-    pagesFor={(obj: ClusterServiceVersionKind) => [
+    pagesFor={(obj: ClusterServiceVersionKind) => _.compact([
       navFactory.details(ClusterServiceVersionDetails),
       navFactory.editYaml(),
+      {
+        href: 'subscription',
+        name: 'Subscription',
+        component: () => <CSVSubscription installedCSV={obj} />,
```

Comment:
Can you check that there are no performance problems declaring this inline? Does this create a new component every render?

Reply:
Switched to using the component directly.